### DISCUSSION TITLE
希望能把设备归类到对应的域下

### DISCUSSION THREAD

**Discussion by yyax** [XiaoMi/ha_xiaomi_home#154] [Ideas]
首先感谢小米公司，感谢相关团队给我们带来了这个集成
我今天试一下该集成，单从ha用户的角度来讲，已经是能用了，但是对于我这种把ha设备桥接进homekit的用户来说，有点不友好。
拿我的浴霸举例，我把设备接入ha后在ha里面得到了select和select域的实体，但是把它桥接进homkit后就变成选择器，基本属于不可用的状态。假如把这个浴霸换成climate域的实体，那在homekit里面就能直接控制了。
![image](https://github.com/user-attachments/assets/198e6756-ed10-4cf6-aa85-6c32b936aa05)
![1d00397305e0178a7c312bb4a356621](https://github.com/user-attachments/assets/c0cab3ed-56c7-47c7-aea7-127ca69ff761)
再次感谢你们！

**Comment by HoTK1** [comment]
同样的还有净化器也是这样

**Reply by yyax** [reply]
嗯，我只是举例说一下，我的马桶都是一大堆选择器，希望越来越完善

**Comment by SusanPhevos** [comment]
请帮忙验证一下 #470 功能是否正常

**Reply by yyax** [reply]
集成版本v0.1.4b1，设备并没有转换，还是和上面图片的一样

**Comment by topsworld** [comment]
后续会慢慢优化，提升用户体验
